src/components/payment/PaymentSection.js

```diff
@@ -154,11 +158,18 @@ class PaymentSection extends React.Component {
             ''
           )}
           {priceGoesUpOn && nextDiscountPrice ? (
-            <P>
-              Ticket price goes up to{' '}
-              {formatPrice(currency, nextDiscountPrice, DEFAULT_VAT_RATE)} on{' '}
-              {priceGoesUpOn}
-            </P>
+            <React.Fragment>
+              <P>
+                Ticket price goes up to{' '}
+                {formatPrice(currency, nextDiscountPrice, DEFAULT_VAT_RATE)} on{' '}
+                {priceGoesUpOn}
+              </P>
+              {showCountDownTimer && (
+                <P>
+                  <Countdown date={countdownDate} />
```

Comment:
@ric9176 new Date(priceGoesUpOn)?

Reply:
:100: just need to refactor so our string is a valid arg to Date() :smile: